src/onnx/onnx_parser.cpp

```diff
@@ -98,7 +98,13 @@ instruction_ref onnx_parser::node_info::add_broadcastable_binary_op(const std::s
                                                                     instruction_ref arg0,
                                                                     instruction_ref arg1) const
 {
-    return add_common_op(*mod, make_op(op_name), {arg0, arg1});
+    return migraphx::add_common_op(*mod, make_op(op_name), {arg0, arg1});
```

Comment:
This can be `this->add_common_op(make_op(op_name), {arg0, arg1})`

Reply:
Done